perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -481,6 +481,13 @@ sub munge_report_new_contacts {
     } else {
         @$categories = grep { grep { $_ ne 'Waste' } @{$_->groups} } @$categories;
     }
+
+    my $user = $self->{c}->user;
+    if ($user && $user->from_body($self->body) && $user->get_extra_metadata('assigned_categories_only')) {
```

Comment:
from_body is a getter/setter, not a check; I think you want `belongs_to_body` (with the body ID, note).

Reply:
Gah I even looked at the definition of `belongs_to_body` to check it did the right thing, but obviously forgot to, you know, actually update the code 🤦‍♂️